parlai/core/utils.py

```diff
@@ -650,8 +650,8 @@ def display_messages(msgs, prettify=False, ignore_fields=''):
             lines.append(msg['image'])
         if msg.get('text', ''):
             text = msg['text']
-            if len(text) > 1000:
-                text = text[:1000] + '...'
+            if len(text) > max_len:
+                text = text[:max_len] + '...'
```

Comment:
in my experience it would actually be nicer to see a bit of the beginning and a bit of the end..

Reply:
good point i'll put out a diff to add that functionality